src/Mirza.Web/Pages/Index.cshtml

```diff
@@ -95,10 +95,32 @@
 
 @section Scripts {
     <partial name="_ValidationScriptsPartial" />
+    <script src="lib/timepicker/tail.datetime-full.min.js"></script>
```

Comment:
It is already obvious that the code imports in the _Layout.cshtml_ should be refactored (e.g. minified, redundants removed, etc.) so I suggest you put this with all those other library imports for a simpler future refactoring flow.

Reply:
Fixed